site/docs/v1/tech/installation-guide/docker.adoc

```diff
@@ -43,12 +43,18 @@ The stock `.env` file will contain the following values, you will want to modify
 ```
 POSTGRES_USER=postgres
 POSTGRES_PASSWORD=postgres
+# Prior to version 1.19.0, using DATABASE_USER
 DATABASE_USER=fusionauth
+# >= 1.19.0, using DATABASE_USERNAME
+DATABASE_USERNAME=fusionauth
```

Comment:
Do we want to leave both values un-commented? I would assume we'd show the new value, and leave the old one commented out.

Reply:
Ah, I was just pulling from https://github.com/FusionAuth/fusionauth-containers/blob/master/docker/fusionauth/.env but could see how having it be backward compatible in the file is different than for documentation. Will make that change.